platform/mds/mds/src/test/java/org/motechproject/mds/it/service/EntityServiceContextIT.java

```diff
@@ -215,6 +215,36 @@ public void shouldSaveEntityWithGivenLookups() throws IOException {
     }
 
     @Test
+    public void shouldAddNewFieldForLookupAndSaveEntity() throws IOException {
+        EntityDto entityDto = new EntityDto();
+        entityDto.setName("myEntity");
+
+        Map<String, Object> values = new HashMap<>();
+        values.put("path", "indexes.0.$addField");
+        values.put("advanced", true);
+        DraftData draftData = new DraftData();
+        draftData.setEdit(true);
+        draftData.setValues(values);
+        entityDto = entityService.createEntity(entityDto);
```

Comment:
please group lines in this test, it's messy. This line with creating new entity should be above creating drafdata